### PR TITLE
Add all transformation modes

### PR DESCRIPTION
**Pull request by mgramin**

### CODE REVIEW

**Review by inponomarev — CHANGES_REQUESTED** [review]
In your example we are using `-f docker-compose.yaml1 together with either  `-f input-db.yaml` or `-f empty-input-db.yaml`

I think this case is a sweet spot for docker-compose [overriding feature](https://docs.docker.com/compose/extends/#adding-and-overriding-configuration)

That is,

* `empty-input-db.yaml` can be incorporated into main docker-compose.yaml
* `input-db.yaml` can go like this:

```yaml
version: '3.5'

services:
  input_db:
    volumes:
      - ./pagila-data.sql:/docker-entrypoint-initdb.d/2.sql
```

And that's it!

Now for empty databases, it's still just `docker-compose run tdk`, for non-empty databases it's `docker-compose -f input-db.yaml run tdk`

Less command line options to type, less code duplication, wdyt?

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -7,10 +7,9 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
 
+  check:
```
[the remark this answers is not in the document]
Comment (mgramin) [reply .github/workflows/check.yml]: Fixed in [7521165](https://github.com/synthesized-io/pagila-tdk-generation/pull/3/commits/7521165964aef006e6d1176a3f53e4f5d547d459)

File: .github/workflows/check.yml [hunk 2]
```diff
@@ -19,28 +18,161 @@ jobs:
...
+          yamllint -sd relaxed .
+
+
+  generation_from_scratch:
+    needs: check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python 3.7
```
[the remark this answers is not in the document]
Comment (mgramin) [reply .github/workflows/check.yml]: Fixed in [7521165](https://github.com/synthesized-io/pagila-tdk-generation/pull/3/commits/7521165964aef006e6d1176a3f53e4f5d547d459)

File: test_tdk.py [hunk 3]
```diff
@@ -2,22 +2,22 @@
 
 @pytest.mark.parametrize("table", [
```
[the remark this answers is not in the document]
Comment (mgramin) [reply test_tdk.py]: Fixed in [7521165](https://github.com/synthesized-io/pagila-tdk-generation/pull/3/commits/7521165964aef006e6d1176a3f53e4f5d547d459)

**Pull request merged**